### PR TITLE
add felds/quick-mailer-bundle recipe

### PR DESCRIPTION
**Pull request by felds**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    from: { name: "Sender Name", email: "sender@example.com" }
+
+    # # Reply-to info (optional)
+    # reply_to: { name: "Reply-to Name", email: "reply-to@example.com" }
+
+    templates:
+        # # Short-form (enabled by default)
+        # demo: path/to/template.html.twig
+
+        # # Long-form
```
Comment (Nyholm) [inline felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: Do you really need both these examples? I think one is enough, the rest could be read in the docs.

File: felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+quickmailer:
+    # Sender info (required)
+    from: { name: "Sender Name", email: "sender@example.com" }
+
+    # # Reply-to info (optional)
+    # reply_to: { name: "Reply-to Name", email: "reply-to@example.com" }
+
+    templates:
+        # # Short-form (enabled by default)
```
Comment (Nyholm) [inline felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: Just write: 

This creates a template named “demo”

**Comment by felds** [comment]
@Nyholm

Those are great tips! Thanks! I'll be using them!

About my last PR, I closed it because I changed something in the package itself and I had no way to re-run the `continuous-integration/symfony/pr` hook without changing something here (i think).

File: felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# For the full configuration, visit:
```
Comment (Nyholm) [inline felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: Could you rename this file to felds_quickmailer.yaml?

Comment (felds) [reply felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: then will we have to change the config root to `felds_quickmailer` too?

Comment (Nyholm) [reply felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: No no. Just the file name

Comment (felds) [reply felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: I find that a little misleading. Are there many packages using a file name different that the config root?
I assume the same conditions that could lead to a filename conflict would also break the config…

Comment (Nyholm) [reply felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: The config root has really nothing to do with this. The file name of the config file (in the recipe) should be prefixed with the organization. 

I see that this is not documented anywhere, but this is a practice done by many (most?) packages.

Comment (felds) [reply felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml]: good to know :)

I'll fix this in a minute

**Comment by Nyholm** [comment]
I see you decided to change the config root anyways. Im fine with that, but it was not necessary.

**Comment by felds** [comment]
-- Felds

**Pull request merged**